src/languages/php/frameworks.md

```diff
@@ -0,0 +1,99 @@
+# PHP Featured Frameworks
+
+Full Drush support and Composer-based builds make handling dependencies and builds for PHP frameworks as simple as committing your `composer.json` to your project.
+
+## Drupal
+
+[Drupal](https://www.drupal.org/) is an open-source content management framework written in PHP. Since Composer comes pre-installed on Platform.sh, Drupal can be installed and updated completely using Composer. The default [build flavor](/overview/build-deploy.md#building-the-application) for PHP application runs `composer install` during build, handling all of your dependencies automatically.
+
+* [Drupal 7 Best Practices](/frameworks/drupal7.md)
+* [Drupal 8 Best Practices](/frameworks/drupal8.md)
+
+### Community Support
+
+Drupal FAQs, how-to guides and other tutorials right on [Platform.sh Community](https://community.platform.sh/).
+
+* [Drupal on Platform.sh Community](https://community.platform.sh/search?q=drupal&expanded=true)
+
+### Templates
+
+* [Drupal 7](https://github.com/platformsh-templates/drupal7)
+* [Drupal 7 (Vanilla)](https://github.com/platformsh-templates/drupal7-vanilla)
+* [Drupal 8](https://github.com/platformsh-templates/drupal8)
+* [Drupal 8 (Multisite variant)](https://github.com/platformsh-templates/drupal8-multi)
+* [Opigno](https://github.com/platformsh-templates/drupal8-opigno)
+* [GovCMS8](https://github.com/platformsh-templates/drupal8-govcms8)
+
+### Example Projects
+
+> **Note:**
+> Template projects (repositories in the `platformsh-templates` GitHub organization) are actively maintained by the Platform.sh team. Any other example projects come with less support, and remain in public repositories as proof-of-concepts.
+
+* [Drupal Commerce 7](https://github.com/platformsh/platformsh-example-drupalcommerce7)
+* [Drupal 8 with Blackfire](https://github.com/platformsh/demo-drupal-blackfire)
```

Comment:
I'm not sure we even need these?  Especially the latter.  Where the heck did that come form?

Reply:
The first is currently listed in PHP: https://docs.platform.sh/languages/php.html#applications. The second one I may have very well github searched "examples" to see if any hadn't made their way into docs.

Reply:
Let's kill both.  Neither one is maintained, and no one in their right mind is starting a D7 commerce site today.